database/sql/V048__create_settlement_rewards.sql

```diff
@@ -0,0 +1,31 @@
+-- Populated before settlement occurs on chain
+-- (after the submissions have been ranked)
+CREATE TABLE settlement_scores (
+  transaction_id bytea NOT NULL,
```

Comment:
Let's change this to use `auction_id bigint PRIMARY KEY`. Remove `transaction_id`.

Reply:
will do.